.eslintrc

```diff
@@ -5,7 +5,13 @@
     "import/no-extraneous-dependencies": [
       "error",
       {
-        "devDependencies": ["config/*.js", "**/*.test.jsx", "**/*.test.js"]
+        "devDependencies": [
+          "babel.config.js",
+          "config/*.js",
+          "**/*.test.jsx",
+          "**/*.test.js",
+          "src/utils/tests.jsx"
```

Comment:
I'm kind of in favor of calling your utils test file `utils.test.jsx`, for consistency, instead of adding this line.

Reply:
hmm that would be nice, except jest now complains that there must be at least one test. I could add a single test with skip in there to appease jest, or add a testName glob pattern in the package.json file. Either of these seems hacky to me... hmm unless I can think of something better I will leave this here for now.. perhaps there is a shorter way to tell jest, this is not a test?